docs/en/built_in_function_develop_guide.md

```diff
@@ -38,12 +38,12 @@ Developers can declare function in [hybridse/src/udf/udf.h](https://github.com/4
 
 #### 2.1.2 C++ Function Naming Rules
 
-- Fuction names are all lowercase, with underscores between words. Check [snake_case](https://en.wikipedia.org/wiki/Snake_case) for more details.
-- Function names would be clear and readable. Use names that describe the purpose or intent of the function.
+- Function names are all lowercase, with underscores between words. Check [snake_case](https://en.wikipedia.org/wiki/Snake_case) for more details.
+- Function names should be clear and readable. Use names that describe the purpose or intent of the function.
 
 #### 2.1.3 C++ and SQL Data Type
 
-C++ built-in function can use limited data types, including BOOL, Numeric, String, Timestamp and Date.The correspondence between the SQL data type and the C++ data type is shown as followed:
+C++ built-in functions can use limited data types, including BOOL, Numeric, String, Timestamp and Date.The correspondence between the SQL data type and the C++ data type is shown as follows:
 
 - | SQL Type  | C++ Type           |
   | :-------- | :----------------- |
```

Comment:
Could you please help use format int16_t to  `int16_t` in the table?

Reply:
@jingchen2222 Good eye, I updated it.